pages/projects/[id]/branches.tsx

```diff
@@ -180,6 +183,14 @@ export const BranchesPage = ({ projects, currentProject }: Props) => {
       projects={projects}
       currentProject={currentProject}
     >
+      <CreateBranchModal
+        onSuccessCreation={() => {
+          utils.branches.getAll.invalidate();
```

Comment:
Curious what `invalidate` will do, i have not checked doc yet, asking for myself to learn more about trpc.

Reply:
![image](https://user-images.githubusercontent.com/3684236/222379155-4f207f23-e1cf-41c6-80df-b4db07312881.png)

Reply:
`invaidate` will invalidate current query data and refetch fresh records